setup.py

```diff
@@ -161,15 +163,22 @@ def protoc(python_out, proto_file, *extra, grpc=True):
         log.info("%s -m grpc.tools.protoc " + " ".join(main_args), sys.executable)
         protoc_module.main(main_args)
 
+    sdk_root = j("bblfsh", "gopkg", "in", "bblfsh", "sdk", SDK_MAJOR)
     # SDK
-    protoc(j("bblfsh", "gopkg", "in", "bblfsh", "sdk", SDK_MAJOR, "protocol"),
+    protoc(j(sdk_root, "protocol"),
            j("gopkg.in", "bblfsh", "sdk." + SDK_MAJOR, "protocol", "generated.proto"),
            "-I" + j("gopkg.in", "bblfsh", "sdk." + SDK_MAJOR, "protocol"))
     # UAST
     protoc("bblfsh", j("github.com", "gogo", "protobuf", "gogoproto", "gogo.proto"),
            grpc=False)
     protoc("bblfsh", j("gopkg.in", "bblfsh", "sdk." + SDK_MAJOR, "uast", "generated.proto"),
            grpc=False)
+    for line in fileinput.input([j(sdk_root, "protocol", "generated_pb2.py"),
```

Comment:
btw check this out how I easily replace in multiple files in Python cc/ @marnovo @gomesfernanda @src-d/machine-learning-team